frontend/package.json

```diff
@@ -10,6 +10,8 @@
     "pikaday": "^1.8.0",
     "autoprefixer": "^9.8.5",
     "http-proxy-middleware": "^1.0.5",
+    "moment": "^2.27.0",
+    "pikaday": "^1.8.0",
```

Comment:
there seems to be more than one pikaday package?

Reply:
You're right! Fixed